pulseapi/entries/migrations/0008_entry_moderation_state.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-05-25 22:57
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from pulseapi.entries.models import ModerationState, Entry, get_default_moderation_state
+
+
+def forwards_func(apps, schema_editor):
```

Comment:
this function basically uplifts all Entries so that if their `is_approved` value is True, their new `moderation_state` value gets updated to point to the "Approved" moderation state